downstream/assemblies/eda/assembly-eda-projects.adoc

```diff
@@ -0,0 +1,9 @@
+[id="eda-projects"]
+
+= Projects
+
+[role="_abstract"]
+
+Projects are a logical collection of rulebooks.
+
+include::eda/proc-eda-set-up-new-project.adoc[leveloffset=-0]
```

Comment:
This should be [leveloffset=+1] which indicates that it is a subsection of this assembly. Then in the procedure module, the title should have only 1 = preceding the title text. I'll comment there as well. I believe nesting was previously managed with the = in the modules but now that we're using assemblies, the nesting is controlled by the leveloffset.

Reply:
Done :)